scripts/objectstore/copy_files_to_irods.py

```diff
@@ -0,0 +1,94 @@
+import os
+import subprocess
+import sys
+import time
+sys.path.insert(0, '../../packages/objectstore/')
```

Comment:
Note to self. This path update is not needed. Must install all Galaxy packages in a venv instead (plus psycopg2)